person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permision: true)
+    @id = Random.rand(1..10)
```

Comment:
- [ **OPTIONAL** ] What do you think if you increase the range of your `id` to something like `100` or `1000` as it can happen that a user create more than person when testing or using your app. But Good :+1: job.